src/quacc/utils/calc.py

```diff
@@ -161,6 +161,18 @@ def run_ase_opt(
     if relax_cell and atoms.pbc.any():
         atoms = ExpCellFilter(atoms)
 
+    if optimizer_kwargs["use_TRICs"] and optimizer.__name__ != "Sella":
+        msg = "Can only use translation rotation internal coordinates aka TRICs with Sella!"
+        raise ValueError(msg)
+    elif optimizer_kwargs["use_TRICs"] and optimizer.__name__ == "Sella":
+        optimizer_kwargs.pop("use_TRICs")
+        from sella import Internals
+        ints = Internals(atoms,allow_fragments=True)
+        ints.find_all_bonds()
+        ints.find_all_angles()
+        ints.find_all_dihedrals()
+        optimizer_kwargs["internal"] = ints
```

Comment:
Any situation where the user would be providing `internal` manually?

Reply:
Yes. Added another conditional to check for this.